We don't have a .git folder/git checkout so this call will fail or do something unexpected.

Author: Alexander Grund (TU Dresden)

diff --git a/setup.py b/setup.py
--- a/python/setup.py
+++ b/python/setup.py
@@ -55,14 +55,6 @@ class BackendInstaller:
             assert backend_name in os.listdir(
                 root_dir), f"{backend_name} is requested for install but not present in {root_dir}"
 
-            try:
-                subprocess.run(["git", "submodule", "update", "--init", f"{backend_name}"], check=True,
-                               stdout=subprocess.DEVNULL, cwd=root_dir)
-            except subprocess.CalledProcessError:
-                pass
-            except FileNotFoundError:
-                pass
-
             backend_src_dir = os.path.join(root_dir, backend_name)
 
         backend_path = os.path.abspath(os.path.join(backend_src_dir, "backend"))
